src/Controller/TagController.php

```diff
@@ -87,13 +87,13 @@ public function addTagFormAction(Request $request, Entry $entry, TranslatorInter
      * Removes tag from entry.
      *
      * @Route("/remove-tag/{entry}/{tag}", name="remove_tag", methods={"GET"}, requirements={"entry" = "\d+", "tag" = "\d+"})
+     * @IsGranted("UNTAG", subject="entry")
+     * @IsGranted("DELETE", subject="tag")
```

Comment:
added this because the tag can be removed completely when no more entries

Reply:
Shouldn't this comment be in the code instead?

Reply:
good idea